tests/v2/validation/snapshot/snapshot_restore_k8s_upgrade_test.go

```diff
@@ -48,14 +48,6 @@ func (s *SnapshotRestoreK8sUpgradeTestSuite) TestSnapshotRestoreK8sUpgrade() {
 		UpgradeKubernetesVersion: "",
 		SnapshotRestore:          "kubernetesVersion",
 		RecurringRestores:        1,
-		ReplaceWorkerNode:        false,
-	}
-
-	snapshotRestoreAll := &etcdsnapshot.Config{
```

Comment:
can you please create + link a follow up ticket in qa-tasks for this / the additional test that we discussed offline

Reply:
Yeah, I left a comment in the issue noting I'm going to follow up with this in the same issue.